### PR TITLE
[-] CORE : Reinstate missing _PS_PRICE_COMPUTE_PRECISION_

### PR DESCRIPTION
**Pull request by xBorderie**
Remake of #2496 for the `1.6.1.x` branch, initially by @christianverardi .

See comments on Greg's 18e4f43d8ff90a66870b86be035c830b8af5134a commit to understand why it was removed in the first place.

### CODE REVIEW

File: controllers/front/ParentOrderController.php [hunk 1]
```diff
@@ -311,11 +311,11 @@ protected function _assignSummaryInformations()
                 $product['id_product'],
                 !Product::getTaxCalculationMethod(),
                 $product['id_product_attribute'],
-                6,
+                _PS_PRICE_COMPUTE_PRECISION_,
```
Comment (maximebiloe) [inline controllers/front/ParentOrderController.php]: ok for this one

File: controllers/front/ParentOrderController.php [hunk 2]
```diff
@@ -311,11 +311,11 @@ protected function _assignSummaryInformations()
...
                 null,
                 false,
                 false,
-                1,
+                6,
```
Comment (maximebiloe) [inline controllers/front/ParentOrderController.php]: I disagree for this one. This is the quantity, please let `1`

**Comment by xBorderie** [comment]
Ok, thanks Maxime, I'll update it as soon as possible, then put it in the milestone myself.

File: controllers/front/ParentOrderController.php (hunk 2 above)
Comment (xBorderie) [inline controllers/front/ParentOrderController.php]: Alright, I'll update it.

**Pull request closed without merging**